sdk/servicebus/azure-messaging-servicebus/migration-guide.md

```diff
@@ -43,24 +43,31 @@ naming, and API structure. Additionally, many developers have felt that the lear
 did not offer a good, approachable, and consistent onboarding story for those learning Azure or exploring a specific
 Azure service.
 
-To improve the development experience across Azure services, including Service Bus, a set of uniform [design
+To try and improve the development experience across Azure services, including Service Bus, a set of uniform [design
 guidelines](https://azure.github.io/azure-sdk/general_introduction.html) was created for all languages to drive a
 consistent experience with established API patterns for all services. A set of [Java specific
 guidelines](https://azure.github.io/azure-sdk/java_introduction.html) was also introduced to ensure that Java clients
 have a natural and idiomatic feel that mirrors that of Java developers. Further details are available in the guidelines
 for those interested.
 
+### Cross Service SDK improvements
+
 The new Service Bus library `azure-messaging-servicebus` provides the ability to share in some of the cross-service
-improvements made to the Azure development experience, such as using the new `azure-identity` library to share a single
-authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each
-of the client libraries.
+improvements made to the Azure development experience, such as 
+ - Using the new `azure-identity` library to share a single authentication between clients. 
+ - A unified diagnostics pipeline offering a common view of the activities across each of the client libraries.
 
 While we believe that there is significant benefit to adopting the new Service Bus library `azure-messaging-servicebus`,
 it is important to be aware that the previous version `azure-servicebus` have not been officially deprecated. They will
 continue to be supported with security and bug fixes as well as receiving some minor refinements. However, in the near
 future they will not be under active development and new features are unlikely to be added to them.
 
-## General changes
+### New features
+
+- Ability to create a batch of messages with the smarter `ServiceBusSenderClient.createMessageBatch()` and 
+`ServiceBusMessageBatch.tryAddMessage()` APIs. This will help manage the messages to be sent in the most optimal way.
```

Comment:
Can we add the below points as new features as well? Please re-word as needed
- Dedicated clients for sync and async operations with the client for async operations using Reactor
- Use of builders to build the client
- Connection poolingCan we add the below points as new features as well? Please re-word as needed
- Dedicated clients for sync and async operations with the client for async operations using Reactor
- Use of builders to build the client


Or maybe these points should go under  Cross Service SDK improvements?
Check with others in the Java team as these are not specific to Service Bus